### DISCUSSION TITLE
Mine craft game

### DISCUSSION THREAD

**Discussion by Cosmic3786** [community/community#34662] [Other Feature Feedback, Questions, & Ideas]
when I try to save a creative world is does not let me

**Comment by PythonCoderAS** [comment]
???

Wrong Microsoft product my guy

**Comment by vicvenpet** [comment]
Try reinstall the game, I don't understand what you mean by "save" anyway. By the way, I think it would be better to ask in a Minecraft forum than here... [Minecraft Reddit](https://www.reddit.com/r/Minecraft/)
